amethyst_assets/src/progress.rs

```diff
@@ -157,7 +158,9 @@ impl Tracker for () {
     fn success(self: Box<Self>) {}
     fn fail(self: Box<Self>, e: Error) {
         error!("error: {}", e);
-        e.iter().skip(1).for_each(|e| error!("caused by: {}", e));
+        while let Some(e) = e.cause() {
```

Comment:
I'm not sure this is correct, it doesn't recurse down the cause chain it just seems to print the same cause over and over again.

Reply:
Yeah I commented the same.

Reply:
Oops thankyou I'll fix, sorry i thought it was correct.